src/components/organisms/AdminDashboard/AdminDashboard.tsx

```diff
@@ -29,10 +30,13 @@ import "./AdminDashboard.scss";
 export const AdminDashboard: React.FC = () => {
   const { ownedVenues, isLoading } = useOwnedVenues({});
 
-  const { worldId } = useWorldEditParams();
+  const { worldSlug } = useWorldEditParams();
 
-  const venues = worldId
-    ? ownedVenues.filter((venue) => venue.worldId === worldId)
+  const { world, isLoaded } = useWorldBySlug(worldSlug);
+
+  console.log(world);
```

Comment:
```suggestion
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2474/commits/810959fc143f367fa1c4c139201437a894fc3b54